ramp-frontend/ramp_frontend/views/ramp.py

```diff
@@ -319,6 +319,25 @@ def sandbox(event_name):
     )
     upload_form = UploadForm(prefix='upload')
 
+    #  check if the event is before, during or after open state
+    now = datetime.datetime.now()
+    start = event.opening_timestamp
+    end = event.closing_timestamp
+
+    event_status = {"msg": "",
+                    "state": "not_yet"}
+    start_str = start.strftime("%d of %B %Y at %H:%M")
+    end_str = end.strftime("%d of %B %Y, %H:%M")
+    if now < start:
+        event_status["msg"] = f"Event submissions will open on the {start_str}"
```

Comment:
Are we supporting f string. Check that we are supporting at least python 3.6 I think.

Reply:
f string works from python 3.6 but I changed it to be sure
Otherwise I think it's ready to go
what do you say @glemaitre ?